aiohttp/web.py

```diff
@@ -289,7 +289,7 @@ async def _run_app(
     host: Optional[Union[str, HostSequence]] = None,
     port: Optional[int] = None,
     path: Optional[str] = None,
-    sock: Optional[socket.socket] = None,
+    sock: Optional[Union[socket.socket, Iterable[socket.socket]]] = None,
```

Comment:
Please import Iterable from `typing`, not `collections.abc`

Reply:
Aw, I did not notice it. Replaced to TypingIterable